README.md

```diff
@@ -234,8 +237,14 @@ it will produce
 it can be expressed like a simple url (http(s)://whateverurl.domain/whateverfile.txt) or it can be expressed using place holders like `https://raw.githubusercontent.com/${GROUP}/${PROJECT}/${BRANCH}/dependencies.yaml` to get `dependencies.yaml` file from source group, project and branch tokens (no need to use all of them at the same time).
 
 **Place holders**
+
 The place holders must be wrapped between `${}` expression and you can use any place holder you want.
 
+**Javascript expressions**
+
+It is possible to define javascript expressions wrapped by `%{}`. So for instance a definition file string like
+`` https://raw.githubusercontent.com/kiegroup/droolsjbpm-build-bootstrap/%{process.env.GITHUB_BASE_REF.replace(/(\\d*)\\.(.*)\\.(.*)/g, (m, n1, n2, n3) => `${+n1+7}.${n2}.${n3}`)}/.ci/pull-request-config.yml `` in case `GITHUB_BASE_REF` environment variable's value is `1.2.x`, the expression between `%{}` will be evaluated and final definition file will be `https://raw.githubusercontent.com/kiegroup/droolsjbpm-build-bootstrap/8.2.x/.ci/pull-request-config.yml`.
```

Comment:
Just out of curiosity, why not enabling the expressions with the `${}` syntax?

Reply:
@rsynek I would prefer to use `${}` as you suggest but, let's suppose a `definition-file` url like ```"https://raw.githubusercontent.com/kiegroup/droolsjbpm-build-bootstrap/${process.env.GITHUB_BASE_REF.replace(/(\\d*)\\.(.*)\\.(.*)/g, (m, n1, n2, n3) => `${+n1+7}.${n2}.${n3}`)}/.ci/pull-request-config.yml"``` the case `${whatever${whatever}}` is found. I can't find the regex to get `${whatever${whatever}}`.
The regex `/\${([^}]+)}/g` will return:

- ${process.env.GITHUB_BASE_REF.replace(/(\\d*)\\.(.*)\\.(.*)/g, (m, n1, n2, n3) => `${+n1+7}
- ${n2}
- ${n3}

Reply:
The other option is to use `${{ }}` and regex `\${{(.*)}}`